recipes/pytest-bandit/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "pytest-bandit" %}
+{% set version = "0.5.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: "40899d30f39af3e8242cb1fd3d086317ef6f9440f3e5370c28f7212b2b15bf28"
+
+build:
+  skip: true  # [py27]
```

Comment:
Looks like this can be `noarch: python` .... that also requires the skip to be removed... instead you can extend the python requirement in host and run to: `- python >3`.